scripts/ci/test:examples

```diff
@@ -10,6 +10,9 @@ if [[ $TRAVIS_COMMIT_MSG == *"[ci disable examples]"* ]]; then
     exit 0
 fi
 
+jupyter notebook --generate-config
+echo "c.NotebookApp.token = u''" >> ~/.jupyter/jupyter_notebook_config.py
```

Comment:
I presume this can't be configured on the command line?

Reply:
This was the Jupyter team's suggested approach. I wish i could do something like ``jupyter notebook --use-token=false`` but that doesn't appear possible.

Reply:
I lied. Looking at their docs, ``jupyter notebook --NotebookApp.token=""`` should work.

Reply:
Nice to know.